models/cow_protocol/ethereum/cow_protocol_ethereum_eth_flow_orders.sql

```diff
@@ -0,0 +1,64 @@
+{{  config(
+        alias='eth_flow_orders',
+        materialized='incremental',
+        partition_by = ['block_date'],
+        unique_key = ['tx_hash'],
```

Comment:
are we confident the tx_hash will be enough to ensure there will be no duplicates in this model?

Reply:
Yes, because users place these orders.... actually... maybe to be a bit safer, we should add `order_uid` in there as well. What do you think?
```suggestion
        unique_key = ['tx_hash', 'order_uid'],
```

Reply:
(just to be clear, there aren't any now and everything lgtm, just want to make sure it won't be a potential issue in the future)

Reply:
adding `order_uid` sounds great 👍

Reply:
Technically I suppose that someone could place an order from a Safe with MultiSend and then there would be two orders in the same tx - I added orderUid just in case.